views/partials/headNavBar.handlebars

```diff
@@ -48,7 +48,7 @@
           {{else}}
           <input type="hidden" name="year" value="{{currYear}}">
           {{/if}}
-          <button type="submit" id="prevMonth" class="btn btn-outline" style="border:
+          <button type="submit" id="prevMonth" class="btn btn-outline" aria-label="Navigate to previous month" style="border:
```

Comment:
good job ! but i think main has these issues fixed you can still cross check it once you update the branch from main